pages/contract_development_and_manufacturing_organization.tsx

```diff
@@ -0,0 +1,86 @@
+import Footer from "../components/Footer";
+
+export default function contract_development_and_manufacturing_organization()
+{
+    
+    return (
+    <>
+        <div className="flex flex-col items-center px-5 px-10 py-5 space-y-10 my-10 page-container" id="aboutus">
+            <h1 className="main-heading">Diversity Bio Sciences: The Ultimate Contract Research Organization (CRO) Offering Specialty Services</h1>
+            <div>
+                
+                <p>
+                Are you looking for the right supplier for custom synthesis of new products? If yes, come straight to Diversity Bio Sciences, a Contract Research Organization (CRO) in Hyderabad that deals in the exclusive synthesis of compounds on behalf of its clients. With us, you can order a certain molecule to be synthesized according to your request, specifications, methods and purities.
+                </p>
+                <p>
+                We have a completely developed, superior quality pharmaceutical impurity standards infrastructure for small-scale custom synthesis of completely new molecules. We enable the outsourcing of different research activities, ranging from the custom synthesis of small molecules, scaffold and drug impurities to the synthesis of intermediates to support on-going medicinal chemistry programs. With our epistemic potentials and skills, we manifest changes in commercial research cultures.
+                </p>
+            </div>
+
+            <div>
+                <h2 className="sub-heading">Full-Service CDMO Also Offering Contract Research and Manufacturing Services</h2>
+                <p>
+                We are a full-service; IAF, EGAC, ISO and QRO certified Contract Development and Manufacturing Organization (CDMO) engaged in the production of thousands of tons of fine chemicals on multi-purpose plants. Our specialties include:
+                </p>
+                <ul className="list-disc space-y-3 text-left text-black block px-10 py-2 rounded-md text-1xl font-normal">

```

Comment:
create a component for <ul/> tag and use that instead.

perhaps

const UL = (props) => {
  return (
    <ul className="list-disc space-y-3 text-left text-black block px-10 py-2 rounded-md text-1xl font-normal">
      {props.children}
    </ul>
  );

}